Dart/src/com/jetbrains/lang/dart/validation/fixes/DartServerFixIntention.java

```diff
@@ -167,7 +169,9 @@ public boolean isAvailable(@NotNull final Project project, final Editor editor,
     final VirtualFile virtualFile = LocalFileSystem.getInstance().findFileByPath(FileUtil.toSystemIndependentName(fileEdit.getFile()));
     final ProjectFileIndex fileIndex = ProjectRootManager.getInstance(project).getFileIndex();
 
-    if (virtualFile == null || !fileIndex.isInContent(virtualFile)) return false;
+    if (fileEdit.getFileStamp() != -1) {
```

Comment:
You're right.
This is another bug in the server.
Fixing it now.
https://codereview.chromium.org/1135073004